Patch already in upstream master:

JENT: C11 - use __asm__
C11 requires the use of __asm__ instead of asm.

Signed-off-by: Stephan Mueller <smueller@chronox.de>

---

diff -aur a/jitterentropy-base-user.h b/jitterentropy-base-user.h
--- a/jitterentropy-base-user.h
+++ b/jitterentropy-base-user.h
@@ -103,7 +103,7 @@
 static inline void jent_get_nstime(uint64_t *out)
 {
 	DECLARE_ARGS(val, low, high);
-	asm volatile("rdtsc" : EAX_EDX_RET(val, low, high));
+	__asm__ volatile("rdtsc" : EAX_EDX_RET(val, low, high));
 	*out = EAX_EDX_VAL(val, low, high);
 }
 
